artemis-jdbc-store/src/main/java/org/apache/activemq/artemis/jdbc/store/drivers/AbstractJDBCDriver.java

```diff
@@ -227,7 +227,11 @@ private void createTableIfNotExists(String tableName, String... sqls) throws SQL
                   }
                }
             } catch (SQLException e) {
-               logger.warn(JDBCUtils.appendSQLExceptionDetails(new StringBuilder("Can't verify the initialization of table ").append(tableName).append(" due to:"), e, sqlProvider.getCountJournalRecordsSQL()));
+               if (logger.isDebugEnabled()) {
+                  logger.debug(JDBCUtils.appendSQLExceptionDetails(new StringBuilder("Can't verify the initialization of table ").append(tableName).append(" due to:"), e, sqlProvider.getCountJournalRecordsSQL()));
+               } else {
+                  logger.infof("Can't verify the initialization of table %s", tableName);
```

Comment:
I think this warrants a LOGGER with ID?

Reply:
which one? debug or infof or both? 
I think it worths to know that something has happened at least...just don't want to scare users for nothing...

Reply:
info...


logger.debug should always be individualized.. .not LOGGER ids for those!

Reply:
uh good point..I haven't thought on it...and trace? I wanted to add more details just if user want...

Reply:
for trace.. it's the same as debug.. although you usually put on if (log.isTraceEnabled()) in front of it... as it's usually a lot more verbose.

The Logger for debug and trace should be:
```java
Logger logger = Logger.getLogger(somethingThatGetsTheClassAndIdon'trememberNow)
if (logger.istTraceEnabled()) {
   logger.trace(....);
}
```

Reply:
I have missed what I should fix on this PR...it is providing a detailed log when DEBUG level is enabled, otherwise will just provide an info. That's because tends DB2 driver to scare users of weird errors when they are not errors at all...

Reply:
@clebertsuconic There is anything I need to do in order to get it merged mate?